packages/ui/Label/stories.tsx

```diff
@@ -0,0 +1,40 @@
+// src/Label.stories.tsx
+import type { Meta, StoryObj } from "@storybook/react";
+import { Label } from "./index";
+
+const meta: Meta<typeof Label> = {
```

Comment:
Please use the consistent approach to type things; other component's stories use `satisfies` to type the meta. Can you please make it the same here?